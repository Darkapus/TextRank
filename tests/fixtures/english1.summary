Users do not rely on storage servers to provide confidentiality nor integrity for their data -- instead all of the data is encrypted and integrity-checked by the gateway, so that the servers can neither read nor modify the contents of the files
