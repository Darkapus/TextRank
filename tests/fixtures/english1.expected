storing or accessing your data
Amazon
store customer data
user's data
data stored
LAFS
